### ISSUE TITLE
Epsilon greedy strategy not working?

### ISSUE THREAD

**Issue by dgilperez** [bmuller/bandit#8]
Hi!

I configured bandit in my production stack, just following the configuration defaults (epsilon_greedy with epsilon: 0.1). I'm using bandit_choose / bandit_convert! methods for alternatives and tracking. After some time, I'm seeing that only one alternative is been shown to the users. Development stack seems to work properly (round_robin strategy).

Short term solution looks simple: switch to round_robin :) But I'd like to know if I missed something in my config or let you know just in case we can track down any bad behaviour with epsilon_greedy.

My bandit.yml below: 

```
default: &DEFAULTS
  player: round_robin
  storage: dalli

development:
  <<: *DEFAULTS

staging:
  <<: *DEFAULTS

test:
  <<: *DEFAULTS

production:
  <<: *DEFAULTS
  player: epsilon_greedy
  player_config:
    epsilon: 0.1
```

Regards!

**Comment by bmuller** [comment]
Hey - are you just reloading the page multiple times, or are you creating a new browser session each time you test?  How many times did you do this?  If you have two alternatives, and you convert with one of them (meaning it has a conversion rate of 100%), then you would only expect to see the other alternative _once_ after _ten_ new browser sessions.  One thing you can do is set the epsilon value to something much higher, like say 0.99, and then you should see the other alternative immediately after one or two new browser sessions.

**Comment by dgilperez** [comment]
I tried using different browsers, cleaning cookies caches on those, and even using different machines. I have 5 alternatives and the first one has 35 participants and 3 conversions, the fourth has 1 participant and the rest 0 participants. I'm not sure if this is consistent with the behaviour intended.

I understand better now how epsilon works. I switched on the morning to round_robin, I'll let the think get some conversions and participants on every alternative and then switch back again to epsilon.

I'll try let you know about the outcomes!
